build/handlers/index.js

```diff
@@ -41,15 +41,18 @@ function exitError(err) {
   process.exit(1)
 }
 
-function fetchSave(resource) {
+function fetchSave(resource, fetchParseFixFn = fetchParseFix) {
```

Comment:
Same comment as in `fetchParseFix`

Reply:
I could of course make manual mocks for these the same way I did for `./build/handlers/save` but I believe then we'd have to move these out into their own module. I don't think it is a huge deal though.